packages/code/src/utils/getTestUtils/getTestUtils.ts

```diff
@@ -55,9 +55,14 @@ export const getTestUtils = (
   const getCopyButtonUtils = () =>
     getButtonTestUtils<HTMLButtonElement>(lgIds.copyButton);
 
-  const getExpandButton = () => {
-    const { getButton } = getButtonTestUtils(lgIds.expandButton);
-    return getButton() as HTMLButtonElement;
+  const getExpandButtonUtils = () => {
+    const { queryButton, getButton, findButton } =
+      getButtonTestUtils<HTMLButtonElement>(lgIds.expandButton);
+    return {
+      getButton: () => getButton(),
+      queryButton: () => queryButton(),
+      findButton: () => findButton(),
+    };
```

Comment:
In theory, I don't think new functions need to be created here and the instances returned from `getButtonTestUtils` can just be returned directly
```
    return {
      getButton,
      queryButton,
      findButton,
    };
```
Or even
```
const getExpandButtonUtils = () => getButtonTestUtils<HTMLButtonElement>(lgIds.expandButton);
```
But the latter would currently also return `isDisabled` (and eventually all that are possible), so if those are just irrelevant, the former makes sense to me.

Reply:
Yeah, I purposely did this because this button will never be disabled

Reply:
Makes sense! The new functions still don't need to be created though, as per the first suggestion. Not a blocker, so will leave it up to your discretion, just a bit cleaner imo.
```suggestion
    return {
      getButton,
      queryButton,
      findButton,
    };
```

Reply:
oh that makes sense!